group/views.py

```diff
@@ -147,3 +147,33 @@ def delete(self, request, user_id):
         except Group.DoesNotExist:
             return Response({"error": "Group not found"},
                             status=status.HTTP_404_NOT_FOUND)
+
+    def post(self, request):
```

Comment:
Can we please separate concerns? Let the message endpoint have it's own view class